code/datums/shuttles.dm

```diff
@@ -194,6 +195,7 @@
 	.["templateName"] = name
 	.["templateShortName"] = short_name
 	.["templateDescription"] = description
+	.["templateTags"] = tags
```

Comment:
without modifying the template modification js (ShipEditor.js, i think), this won't do anything

Reply:
Ship editor is fully supported now.